README.md

```diff
@@ -2,70 +2,135 @@
 
 [![mkdocs](https://github.com/Geoff1900/software-development-standards/actions/workflows/publish.yml/badge.svg)](https://github.com/GIG-Cymru-NHS-Wales/software-development-standards/actions/workflows/publish.yml)
 
-This documents in this repository are published to [https://geoff1900.github.io/software-development-standards](https://geoff1900.github.io/software-development-standards/)
-
+The documents in this repository are published to [https://geoff1900.github.io/software-development-standards](https://geoff1900.github.io/software-development-standards/).
```

Comment:
Once you’ve enabled GitHub Pages for the main (upstream) repo, could you update this URL to point to that site rather than my local fork? Thanks!